### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by xanaxsys** [UnamSanctam/UnamWebPanel#202]
Hi, will the webpanel get any updates anymore?

**Comment by UnamSanctam** [comment]
Yes, I've made quite a huge amount of performance upgrades and more, I'm adding the "Hide offline miners" switch right now.

**Comment by xanaxsys** [comment]
Oh insane I'm hyped

**Comment by xanaxsys** [comment]
Any atm then it get released?

**Comment by xanaxsys** [comment]
> Yes, I've made quite a huge amount of performance upgrades and more, I'm adding the "Hide offline miners" switch right now.

Then is release date?

**Comment by UnamSanctam** [comment]
Not sure, I'm trying to implement some other ideas people have had though I've also scrapped some. Right now I'm trying to effectively implement geo flags for the IPs.

**Comment by xanaxsys** [comment]
Sound great
